### ISSUE TITLE
Opera -media indicator overlaps headline on small widths

### ISSUE THREAD

**Issue by dr3** [bbc/simorgh#2337]
**Describe the bug**
At small widths, on opera with a large font, the media indicator overlaps the headline

**Expected behavior**
They do not overlap

**Screenshots**
![image](https://user-images.githubusercontent.com/11341355/60814475-93953280-a18d-11e9-95b4-8002448e865a.png)

**Desktop (please complete the following information):**

- OS: MacOs latest
- Browser: Opera
- Version: 55.0

**Additional context**

**Comment by amywalkerdev** [comment]
Opera 55 is a year old, don't think we support this? Check with newer versions of opera and no smaller than 240px to see if we can recreate.

**Comment by andrew-nowak** [comment]
Opera v60.0.3255.83 (latest macos opera build at writing)

Font size: medium
![image](https://user-images.githubusercontent.com/10963046/60882858-a452b080-a240-11e9-88d7-a897f31d69a3.png)

Font size: large
![image](https://user-images.githubusercontent.com/10963046/60883081-22af5280-a241-11e9-844b-d6fbcd1fc233.png)

Font size: very large
![image](https://user-images.githubusercontent.com/10963046/60883003-f8f62b80-a240-11e9-92bc-6b6d3a9e2ad0.png)

Large headline & font size: very large
![image](https://user-images.githubusercontent.com/10963046/60883125-4377a800-a241-11e9-9cac-e6638facb0fc.png)

looks good to me

**Comment by ghost** [comment]
Should we close this issue?

**Comment by EinsteinNjoroge** [comment]
@aacn500 @clareevansbbc I reproduced this on Opera and Chrome, this only happens when you set a super large custom font size. I think we should close this issue.

**Comment by andrew-nowak** [comment]
What screen width? We don't support anything under 240px.
What Opera and Chrome versions? I couldn't replicate with the latest, but recent versions should be (imo) supported (if not overly complex to do).

We definitely want to support custom font sizes for accessibility, and I'd like to be certain that this isn't an issue before closing.

**Comment by EinsteinNjoroge** [comment]
@aacn500 you can reproduce this by setting a large custom font and use a screen size >550px

**Comment by EinsteinNjoroge** [comment]
@aacn500 ![ezgif com-video-to-gif](https://user-images.githubusercontent.com/21374761/61040972-48bc2a80-a3da-11e9-84e9-277142e8a383.gif)

**Comment by andrew-nowak** [comment]
thanks @EinsteinCarrey , I can replicate what you see now.

I _believe_ (although it would be good to check with @clareevansbbc @greenc05 ) that we only support the 'default' font sizes (eg. 'very large' on the 'font size' drop down menu; and not using the 'customise fonts' slider)

**Comment by clareevansbbc** [comment]
I agree that we don't need to support the 'customise fonts' slider, but worth @greenc05 confirming.

**Comment by greenc05** [comment]
I think this is acceptable, please close
